core/templates/dev/head/moderator/Moderator.js

```diff
@@ -17,38 +17,53 @@
 */
 
 oppia.controller('Moderator', [
-    '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
-    function($scope, $http, $rootScope, oppiaDatetimeFormatter) {
-  $scope.getDatetimeAsString = function(millisSinceEpoch) {
-    return oppiaDatetimeFormatter.getLocaleAbbreviatedDatetimeString(
-      millisSinceEpoch);
-  };
+  '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter', 'alertsService',
+  function($scope, $http, $rootScope, oppiaDatetimeFormatter, alertsService) {
+    $scope.getDatetimeAsString = function(millisSinceEpoch) {
+      return oppiaDatetimeFormatter.getLocaleAbbreviatedDatetimeString(
+        millisSinceEpoch);
+    };
 
-  $scope.getExplorationCreateUrl = function(explorationId) {
-    return '/create/' + explorationId;
-  };
+    $scope.getExplorationCreateUrl = function(explorationId) {
+      return '/create/' + explorationId;
+    };
 
-  // TODO(sll): Abstract this out into a general utility function so that it
```

Comment:
Ah, the TODO was so generic I didn't realize it was referring to the page specifically. That makes more sense.